ccd-definition/CaseField.json

```diff
@@ -279,5 +279,46 @@
     "FieldType": "FixedRadioList",
     "FieldTypeParameter": "ResponseIntention",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "respondentResponseType",
```

Comment:
knowing there is going to be 2v1 and 1v2 in the future, just naming these fields with respondent is not enough

Reply:
I will leave the naming decision to @AlisonRevittHM

Reply:
For this attribute, respondent1ClaimResponseType

Reply:
done